specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/storage.json

```diff
@@ -4040,6 +4070,18 @@
             "modelAsString": true
           },
           "description": "Restrict copy to and from Storage Accounts within an AAD tenant or with Private Links to the same VNet."
+        },
+        "dnsEndpointType": {
+          "type": "string",
+          "enum": [
+            "Standard",
+            "AzureDnsZoned"
```

Comment:
Please change this to AzureDnsZone instead of AzureDnsZoned

Reply:
@amoldixitms - can we get this change done.

Reply:
this was already done, you are looking at an outdated comment

Reply:
Should `dnsEndpointType` added for `StorageAccountPropertiesUpdateParameters`? It seems not updatable per my test